documentation/sample_queries.md

```diff
@@ -54,6 +54,51 @@ where service in ('Storage')
 group by 1, 2, 3, 4
 ```
 
+### Unused Tables
```

Comment:
note: you could include these as "analysis" queries in the analyses folder

Reply:
we could, but planning to move these exapmles to a different site anyways. plus we'd lose the ability to have commentary mixed with code.